.github/workflows/upstream.yml

```
@@ -0,0 +1,20 @@
+name: upstream
+
+on:
+  schedule:
+    - cron: '30 7,19 * * *'
+  workflow_dispatch:
+
+jobs:
+  versioncheck:
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: ubuntu-latest
    if: github.repository_owner == "daviddrysdale"
```

This restricts the scheduled job to just this repo and not forks, which saves annoyance -- and also the check isn't really useful on forks.

A